.github/workflows/CI.yml

```diff
@@ -80,6 +60,24 @@ jobs:
             -DCMAKE_BUILD_TYPE=${{ matrix.cmake_build_type }}
           cmake --build build --parallel 2
           cmake --install build
+      - name: Checkout hypre
+        uses: actions/checkout@v2.2.0
+        with:
+          repository: hypre-space/hypre
+          # FIXME: use tag when https://github.com/hypre-space/hypre/pull/423 is included in a release
+          ref: ae8bbf619ce243e9dab095a8e115717463d3db83
```

Comment:
Is this the latest `master` hash?

Reply:
Yes - I figured this was better than just using master